pinot-segment-local/src/main/java/org/apache/pinot/segment/local/recordtransformer/CompositeTransformer.java

```diff
@@ -104,10 +104,6 @@ public static CompositeTransformer getDefaultTransformer(TableConfig tableConfig
   public static CompositeTransformer composeAllTransformers(List<RecordTransformer> customTransformers,
```

Comment:
This is the same as `getDefaultTransformer()`. Consider simply removing it

Reply:
No, right? cause it also includes custom transformers (although we pass an empty list)